cmd/goreleaser/internal/configure.go

```diff
@@ -30,9 +30,14 @@ import (
 const ArmArch = "arm"
 
 var (
-	ImagePrefixes = []string{"otel", "ghcr.io/open-telemetry/opentelemetry-collector-releases"}
-	Architectures = []string{"386", "amd64", "arm", "arm64", "ppc64le", "s390x"}
-	ArmVersions   = []string{"7"}
+	ImagePrefixes      = []string{"otel", "ghcr.io/open-telemetry/opentelemetry-collector-releases"}
+	Architectures      = []string{"386", "amd64", "arm", "arm64", "ppc64le", "s390x"}
+	ArmVersions        = []string{"7"}
+	DefaultConfigDists = map[string]bool{"otelcol": true, "otelcol-contrib": true}
+	MSIWindowsDists    = map[string]bool{"otelcol": true, "otelcol-contrib": true, "otelcol-otlp": true}
```

Comment:
using const for this dist names wherever we can would be good in case we change the names in the future.

Reply:
are you saying define `otelcol`, `otelcol-contrib`, etc. as consts and then refer to those in the slices or maps here?

Reply:
Yes